lib/aliquot-pay/util.rb

```diff
@@ -11,18 +11,18 @@ def self.generate_shared_secret(private_key, public_key)
       private_key.dh_compute_key(public_key)
     end
 
-    def self.derive_keys(ephemeral_public_key, shared_secret, info)
+    def self.derive_keys(ephemeral_public_key, shared_secret, info, length: 32)
```

Comment:
Thought: What about supplying the "EC version" rather than `length`? I think that could perhaps be more user friendly?

Reply:
Hmm. Sure, that makes sense. :+1: